README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents:
+
+- [Azul Overview](#1-architecture-overview)  
+- [Getting Started](#2-getting-started)
+- [Contributing Guidelines](CONTRIBUTING.rst)
+  
+Request Access to Cloud Infrastructure and Tools: 
+  - Contact the Team Lead or Project Manager for access to the Azul GitHub Repository. 
+  - [AWS Credentials](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+  - [GCP Credentials](#232-google-credentials)
+  - Development [CI/CD systems](#91-cicd-systems)
+  - [Slack](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
```

Comment:
I might move slack higher up the list since it may be necessary for getting access to the others. Also it may be helpful to mention the channel by name #team-boardwalk :shrug: